src/bp/sdk/botpress.d.ts

```diff
@@ -1412,7 +1412,7 @@ declare module 'botpress/sdk' {
     /**
      * Returns an existing user or create a new one with the specified keys
      */
-    export function getOrCreateUser(channel: string, userId: string): GetOrCreateResult<User>
+    export function getOrCreateUser(channel: string, userId: string, botId?: string): GetOrCreateResult<User>
```

Comment:
@allardy so we add the `botId` on channel_users after all 😆

Reply:
Nop, it's only to track them in a separate table (`bot_chat_users`). It doesn't change the structure of srv_channel_users